docs/content/latest/troubleshoot/cluster/replace_tserver.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Replace a failed YB-TServer
+headerTitle: Replace a failed YB-TServer
+linkTitle: Replace a failed YB-TServer
+description: Procedure to replace a failed YB-TServer
+aliases:
+  - /troubleshoot/cluster/replace-tserver/
+  - /latest/troubleshoot/cluster/replace-tserver/
```

Comment:
Remove these -- Aliases are not needed for new pages, except when there are tabbed buttons (then discuss with me).

Reply:
removed